java/code/src/com/suse/manager/webui/services/impl/SaltService.java

```diff
@@ -763,13 +763,13 @@ public void refreshPillar(MinionList minionList) {
         try {
             LocalCall<Boolean> call = SaltUtil.refreshPillar(Optional.empty(),
                     Optional.empty());
-            callAsync(call, minionList);
+            callSync(call, minionList);
```

Comment:
Is this change affects Salt Minions also?
For Salt Minions async calls are possible, but not for Salt SSH clients. Maybe I miss the condition somewhere, but anyway it's better to ensure.

Reply:
yes, it affects also salt minion: having an async call for salt minion and sync call for salt ssh it's possible but I don't know if it would overengineer it...waiting a feeback from @lucidd about that :)

Reply:
It could drop performance dramatically on performing sync calss to the minions. For Salt SSH async calls are not implemented on the salt side, so we have to use it in sync.

Reply:
After talking with @lucidd , we decided to revert this part and create an ad-hoc issue for this async issue